.github/workflows/pipeline-deploy-app-test.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
     needs: [
       deploy-application-infrastructure,
     ]
-    uses: NHSDigital/uec-dos-management/.github/workflows/build-and-deploy-react-app.yaml@v0.18
+    uses: NHSDigital/uec-dos-management/.github/workflows/build-and-deploy-react-app.yaml@v0.23
```

Comment:
should this now only point to the deploy yaml

Reply:
Yes but not sure if Pira is plannikng to do that under next ticket

Reply:
That's right as Tim mentioned planning on doing this as part of DR-928